zlp-scheduler/coverage/.last_run.json

```diff
@@ -1,5 +1,9 @@
 {
   "result": {
+<<<<<<< HEAD
+    "covered_percent": 26.06
+=======
     "covered_percent": 13.75
+>>>>>>> 8781c5fe8a5ec29ca88e8607042daa6617209fb0
```

Comment:
You'll need to remove the lines with the `<<<`, `===`, and `>>>`. And you'll need to make sure that there's only one `"covered_percent"` line. Otherwise it'll throw some errors. You might check the `index.html` and `.resultset.json` files for similar errors too.

Reply:
ok